plugins/draw-tools.js

```diff
@@ -365,7 +375,7 @@ window.plugin.drawTools.optCopy = function() {
       if (stockWarnings.unknown) stockWarnTexts.push('Warning: UNKNOWN ITEM TYPE');
 
       var html = '<p><a onclick="$(\'.ui-dialog-drawtoolsSet-copy textarea\').select();">Select all</a> and press CTRL+C to copy it.</p>'
-                +'<textarea readonly onclick="$(\'.ui-dialog-drawtoolsSet-copy textarea\').select();">'+localStorage[window.plugin.drawTools.KEY_STORAGE]+'</textarea>'
+                +'<textarea readonly onclick="$(\'.ui-dialog-drawtoolsSet-copy textarea\').select();">'+window.localStorage[window.plugin.drawTools.KEY_STORAGE]+'</textarea>'
```

Comment:
I have such concerns about original code:
1. Too looong line.
2. Crazy nesting of quotes
3. Using `onclick` attribute instead of attaching listener in js.

Latter is not so easy to fix though.
But in my opinion, that should be done anyway to make draw tools menu extendable by other plugins.
As you remember, DrawToolsPlus used to rewrite major parts of original code just to add a couple items to that menu.

Reply:
Btw, I see no sense in changing `localStorage` to `window.localStorage`. Former was easier to read.

Reply:
1) add a line break
2) agree, line was not changed in that perspective
3) see 2)

My task was NOT code optimization, but to get rid of DTplus. So just adding it's code where it belongs.

"window.": just trying to follow the way the rest of the code was done. IMHO if you try to understand the code  and suddenly this is dropped you ask yourself "why did the DEV do so?" See my next comment below.

Reply:
> "window.": just trying to follow the way the rest of the code was done.

1. But changing that line was not actually in need for your DTPlus, right?
2. Make all the code follow one style is good goal. Just keep it separately. And choose best patterns (here: `localStorage` withou `window`)

Reply:
> IMHO if you try to understand the code and suddenly this is dropped you ask yourself "why did the DEV do so?"

In fact if one try to understand iitc code currently - he should prepare to real quest with all those legacy patterns everywhere.
Yeah, in future we should try to make it follow some common (and good) style.